test/sequential/test-http-pipeline-flood.js

```diff
@@ -72,20 +88,13 @@ function child() {
   req = new Array(10241).join(req);
 
   conn.on('connect', function() {
+    //kill child after flooding
+    setTimeout(function() { conn.destroy(); }, common.platformTimeout(1000));
```

Comment:
Maybe a better approach is to `child.kill()` in the parent once flood prevention kicks in?